int16 u
int16 v
---
bool success
geometry_msgs/PoseStamped coor_wrt_camera